vaadin-combo-box-flow-parent/vaadin-combo-box-flow/src/main/java/com/vaadin/flow/component/combobox/ComboBoxBase.java

```diff
@@ -145,6 +145,7 @@ public <TValueProperty> ComboBoxBase(String valuePropertyName,
         // wrapper items for identification
         getElement().setProperty("itemValuePath", "key");
         getElement().setProperty("itemIdPath", "key");
+        getElement().setProperty("itemLabelPath", "label");
```

Comment:
BTW, we probably should update the web component to set `label` value by default for consistency.